ingestor/chalicelib/gtfs/ingest.py

```diff
@@ -99,6 +109,7 @@ def ingest_feed_to_dynamo(
                     "lineId": total.line_id,
                     "count": total.count,
                     "serviceMinutes": total.service_minutes,
+                    "hasServiceExceptions": total.has_service_exceptions,
```

Comment:
The "recovery dash" code needs to know, in general, whether the service levels on a given day include any service exceptions (usually additions/removals for holidays). It was easiest just to add this as a column to the `ScheduledServiceDaily` table — the migration has already run.